### DISCUSSION TITLE
USDT in TON network

### DISCUSSION THREAD

**Discussion by Sarets** [horizontalsystems/unstoppable-wallet-android#7665] [Q&A] [answered]
Hello. 

I have sent USDT from Binance via TON network to Unstoppable wallet.

Currently i can't add USDT token in Unstoppable network via TON blockchain. So, I can't send to anyone USDT or even can't see this transaction in Unstoppable wallet.

May i ask you if you have in plans add support of USDT token in TON network?

Or maybe somebody know how to convert BIP39 mnemonic to TON mnemonic?

**Comment by ealymbaev** [comment, marked as answer]
@Sarets we are preparing full support of Jettons for Ton blockchain. It will be released in upcoming version.
